packages/deployment-service-ui/src/components/pages/pipelines/show.tsx

```diff
@@ -1,18 +1,56 @@
 import Routes from '@/constants/routes'
 import { reapitConnectBrowserSession } from '@/core/connect-session'
-import { pipelineServiceGet } from '@/platform-api/pipelines'
+import { pipelineRunnerCreate, pipelineRunnerPaginate, pipelineServiceGet } from '@/platform-api/pipelines'
 import { ReapitConnectSession, useReapitConnect } from '@reapit/connect-session'
-import { Breadcrumb, BreadcrumbItem, FlexContainerBasic, H1, Section } from '@reapit/elements-legacy'
-import { Label, Loader } from '@reapit/elements'
+import { Breadcrumb, BreadcrumbItem, FlexContainerBasic, H1, Section, H3 } from '@reapit/elements-legacy'
+import { Button, Label, Loader, StatusIndicator, Table } from '@reapit/elements'
 import React, { useEffect, useState } from 'react'
 import { Link, useParams } from 'react-router-dom'
-import { PipelineModelInterface } from '@reapit/foundations-ts-definitions'
+import {
+  DeploymentStatus,
+  PipelineModelInterface,
+  PipelineRunnerModelInterface,
+  TaskModelInterface,
+} from '@reapit/foundations-ts-definitions'
+import { Pagination } from 'nestjs-typeorm-paginate'
+import { Intent } from '@reapit/elements/src/helpers/intent'
```

Comment:
You should just be able to. export this from @reapit/elements - if not, it's a bug as helpers are not being exported from the main entry point of elements